x-pack/plugins/security_solution/cypress/integration/timelines/fields_browser.spec.ts

```diff
@@ -104,12 +104,12 @@ describe('Fields Browser', () => {
         });
     });
 
-    it.skip('displays a count of only the fields in the selected category that match the filter input', () => {
+    it('displays a count of only the fields in the selected category that match the filter input', () => {
       const filterInput = 'host.geo.c';
 
       filterFieldsBrowser(filterInput);
 
-      cy.get(FIELDS_BROWSER_SELECTED_CATEGORY_COUNT).should('have.text', '5');
+      cy.get(FIELDS_BROWSER_SELECTED_CATEGORY_COUNT).should('have.text', '4');
```

Comment:
@madirey this change is not correct, we are still having 5 fields, this test was catching a legitimate bug.

The problem here is that when we are opening the timeline the new alert's index is not selected by default when it should, that index is the one that contains the missing field.

We should fix the issue and change the number of the test to 5 again, I'll open a ticket with the bug and assign to the alerts team since I'm assuming that as the issue was introduced when the rule registry changes were merged is the alerts team the one that should take care of it. Please note that this change impacts one of new explore team functionalities.

ping @spong @rylnd @stephmilovic

Reply:
@MadameSheema I only see 4 fields in the ECS field mapping. Is there an additional non-ECS one?

Reply:
https://www.elastic.co/guide/en/ecs/current/ecs-geo.html

As you can see, the test is looking at geo fields that start with c. there are 5

You are only looking at ecs mapping for Data View before it has alerts mappings because alerts index has not been initialized yet. Alerts index has extra field, if i recall correctly `geo.continent_code`

Reply:
@stephmilovic @MadameSheema The new alerts mapping doesn't contain the `geo.continent_code` field so I believe it will only show up on upgrades (since the field exists in the legacy .siem-signals index, which is aliased). New installs will have only 4 fields. Our team will be updating the ECS mappings soon, which should result in that field being added. We should be able to update the test at that point.